jax/experimental/export/_export.py

```diff
@@ -921,6 +967,13 @@ def expand_in_shardings(in_shardings: Sequence[LoweringSharding],
     all_in_shardings[idx] = in_s
   return tuple(all_in_shardings)
 
+def _hlo_sharding_to_xla_compatible_sharding(
+    hlo_sharding: Sharding,
```

Comment:
xc.HloSharding?

Reply:
See the definition of Sharding at the top of the module: xc.HloSharding | None.

Reply:
I saw that but being explicit here is helpful. Maybe you should get rid of Sharding and just use `xc.HloSharding | None` everywhere?

It's confusing to call it in_shardings as it is. Up to you though :)